mllib/src/test/scala/org/apache/spark/mllib/tree/LossesSuite.scala

```diff
@@ -0,0 +1,22 @@
+package org.apache.spark.mllib.tree
```

Comment:
Huh, good point.  It must have been left over after some other change.  Perhaps we should just remove the whole class.